src/banner/banner.story.ts

```diff
@@ -0,0 +1,48 @@
+import { storiesOf } from '@storybook/html';
+import { html, render } from 'lit-html';
+
+
+import notes from './banner.md';
+
+storiesOf('BannerElement', module)
+  .addParameters({
+    notes
+  })
+  .add('Simple', () => {
+
+    const rootElm = document.createDocumentFragment();
+
+    const clickHandler = () => { render(template({open: false}), rootElm)}
+
+    const template = (data: any) => html`
+      <div>
+        <wf-banner .openBanner='${data.open}'>
+          <span slot="bannerContent">Banner Text</span>
+          <button slot="bannerAction">Dismiss<button>
```

Comment:
@sagar735 Is this one button or multiple buttons? Or is it improper closing of tags?